Project.toml

```
@@ -1,6 +1,6 @@
 name = "ChainRules"
 uuid = "082447d4-558c-5d27-93f4-14fc19e9eca2"
-version = "1.69.0"
+version = "1.69.1"
```

As in https://github.com/JuliaDiff/ChainRules.jl/pull/506, I think from the point of ChainRules this is a feature, not a bug fix:

```suggestion
version = "1.70.0"
```